src/main/java/step3/RacingCar.java

```diff
@@ -0,0 +1,15 @@
+package step3;
+
+import java.util.Random;
+
+public class RacingCar {
+
+    private static Random random = new Random();
+    private int moveCount;
+
+    public int move() {
+        int temp = random.nextInt(10);
+        if (temp >= 4) this.moveCount += temp;
```

Comment:
```suggestion
        int randomNumber = random.nextInt(10);
        if (randomNumber >= 4) {
            this.moveCount++;
        }
```
랜덤한 숫자가 4이상일때 한칸 전진하는 방식입니다. 🙂